packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -478,22 +478,23 @@
   padding: 0;
   border: 0;
   background-color: transparent;
-  color: $bpk-link-color;
   text-decoration: $bpk-link-text-decoration;
   cursor: pointer;
   appearance: none;
 
+  @include prop(color, secondary-color);
```

Comment:
Should this really be the secondary color?

Reply:
I used secondary color, cause we use the primary one for the "call-to-action" color (bpk green).

Reply:
Yeah you're right our color scheme is confusing. We might need more variables than Primary and Secondary in the end